Makefile

```diff
@@ -103,6 +109,15 @@ woke: woke-install
 pa11y: pa11y-install html
 	find $(BUILDDIR) -name *.html -print0 | xargs -n 1 -0 $(PA11Y)
 
+vale: install
+	@. $(VENV); test -d $(SPHINXDIR)/venv/lib/python3.12/site-packages/vale || pip install vale
```

Comment:
We don't require 3.12 anywhere afaik ... I just checked, and for me locally it's 3.10.
It might work with a wild card since we're inside the environment (where we should have only one Python version)?